Intersect.Server/Entities/Player.cs

```diff
@@ -4612,13 +4615,25 @@ public void CancelTrade()
         //Parties
         public void InviteToParty(Player fromPlayer)
         {
+            if (fromPlayer == null)
+            {
+                return;
+            }
+
             if (Party.Count != 0)
             {
                 PacketSender.SendChatMsg(fromPlayer, Strings.Parties.inparty.ToString(Name), ChatMessageType.Party, CustomColors.Alerts.Error);
 
                 return;
             }
 
+            if (Options.Instance.Instancing.BlockPartyInvitesInInstance && (!InOpenInstance || !fromPlayer.InOpenInstance))
```

Comment:
I think this should only take into account the inviter, but I have some other thoughts that I am going to include in the more general review comment.